nvflare/app_opt/tf/job_config/model.py

```diff
@@ -12,22 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import Optional
+
 import tensorflow as tf
 
 from nvflare.app_opt.tf.model_persistor import TFModelPersistor
+from nvflare.job_config.api import validate_object_for_job
 
 
 class TFModel:
-    def __init__(self, model):
-        """TensorFLow model wrapper.
+    def __init__(self, model, persistor: Optional[TFModelPersistor] = None):
+        """TensorFlow model wrapper.
 
         If model is a tf.keras.Model, add a TFModelPersistor with the model.
 
         Args:
             model (any): model
+            persistor (Optional[TFModelPersistor]): A TFModelPersistor,
+            if provided will ignore argument `model`, defaults to None.
         """
         self.model = model
 
+        validate_object_for_job("persistor", persistor, TFModelPersistor)
```

Comment:
we should check `if persistor is not None:` before validating

Reply:
good catch, let me fix it.